### PR TITLE
add old versions of ideal-image-slider

### PR DESCRIPTION
**Pull request by kennynaoh**
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/gilbitron/Ideal-Image-Slider
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/ideal-image-slider
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1554
   - Count of watchers: 70
   - Count of forks: 162
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 9
   - Downloads in the last month: 36

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #8065

### CODE REVIEW

**Comment by x09326** [comment]
The first line of commit message is more than 50 chars.
Please modify it.

**Comment by kennynaoh** [comment]
@x09326 
I have changed the commit message.
Please help me review it again.
Thanks!

File: ajax/libs/ideal-image-slider/package.json [hunk 1]
```diff
@@ -14,15 +14,14 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gilbitron/Ideal-Image-Slider.git",
+    "basePath": "",
+    "files": [
+      "ideal-image-slider*",
+      "extensions/**/*.!(md)",
+      "themes/**/*"
+    ]
+  }
```
Comment (PeterDaveHello) [inline ajax/libs/ideal-image-slider/package.json]: There is no need to transfer to git auto-update in the config, you are just adding the missing old versions.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@kennynaoh please don't generate map files to make the inconsistency, v1.0.0 has map file for js but v1.4.0 don't (may be officially minified?), we try to keep the consistency.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
There is a officially minified js file for all the version except v1.0.0 and v1.0.1.
So should I suggest author to provide or the other steps?
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh No, I already told you the solution.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
You mean I don't need to minify in v1.0.0 and v1.0.1?
Thanks!

**Comment by PeterDaveHello** [comment]
"do not generate the map file" isn't this clear?

**Comment by kennynaoh** [comment]
OK, thanks.

**Comment by kennynaoh** [comment]
@x09326 
I have minified the js file without map file in v1.0.0 and v1.0.1.
Please help me review it again.
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh you should also mention the reason why to manually add them in the comment but not only in commit message next time.

**Pull request merged**